Initializes "Content Implies User Intent" blog (#262)
readthedocs/ethicalads.io: content/posts/2022-content-as-intent.md

Title: Does Content Imply User Intent?
Date: April 26, 2022
description: Using content targeting to provide users with relevant ads capitalizes on the same underlying reason that search ads perform so well, intent.
tags: content-targeting, advertisers, marketing, intent
authors: Ra Cohen
image: /images/posts/2022-content-as-intent.jpg
image_credit: <span>Photo by <a href="https://unsplash.com/@jeshoots?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">JESHOOTS.COM</a> on <a href="https://unsplash.com/s/photos/confused?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a></span>

When was the last time you clicked on an ad while browsing social media?
While scrolling through your feed of friends and family, ads for random gadgets or cheesy games annoy many, but why?
The ad itself is undoubtedly tailored to your own personal interests or brands that you like. So why is it irritating?
Because you as a user, had no intention of shopping. Your purpose in logging on was to socialize, not to shop.

## Intent is irreplacable

> Search ads have been the dominant form of display ad since they were invented around 20 years ago and to no wonder.

Reviewer: ```suggestion
Search ads have been the dominant form of display ad since they were invented over 20 years ago and to no wonder.
```

If a user is actively searching for a flower store, there is already an intent to purchase flowers present which motivated this search.
The same can be true of software products and solutions. If a developer is using a search engine to find a cloud hosting service,
chances are good that they intend to utilize a cloud hosting service in the near future.
**User Intent** is the [greatest individual factor](https://www.crazyegg.com/blog/guide-user-intent/) to convert a potential candidate into a paying customer.

But search engines are not the only places where software developers exhibit their intent.

> From using online tools like js.bin rendering their html edits in real-time to scouring package documentation to fix their latest bug,

Reviewer: ```suggestion
From using online tools like [https://jsbin.com](jsbin) rendering their html edits in real-time to scouring package documentation to fix their latest bug,
```

As you can see, almost every week our geo-only campaigns fall below our network average CTR of 0.1%.

<div class="postimage text-center">
  <img class="w-75" src="{static}../images/posts/2022-no-keywords-ctr.png" alt="A screenshot of the Click through Rates for untargeted campaigns">
  <p>the average Click through Rates (grouped by week) for untargeted campaigns ending in the last 5 months</p>
</div>

But how do our content-targeted campaigns fare? Using the same weekly aggregation, it's evident that our content-targeted campaigns frequently exceed our network's average CTR.

<div class="postimage text-center">
  <img class="w-75" src="{static}../images/posts/2022-keywords-ctr.png" alt="A screenshot of the Click through Rates for content-targeted campaigns">
  <p>the average Click through Rates (grouped by week) for content-targeted campaigns ending in the last 5 months</p>
</div>

Our CTR (click through rate) is consistently higher on campaigns which employ content-targeting in addition to geographic targeting.
Which leads one to wonder, given that geography is a factor of the user whereas content is a factor of the publishing website,

> why has so much of ad-tech been focusing on user-targeting and [surveillance advertising](https://www.ethicalads.io/surveillance-advertising/?ref=content-as-intent) as opposed to content-targeting?

Reviewer: Usually you don't use `ref` params on an internal link since an analytics solution should be able to figure that out. Obviously it won't do that if somebody clicks through from the email syndication. I don't feel that strongly, but figured I'd mention it.

Author: Yeah, it was for email inclusivity.

## The Return of Content Targeting

Content Targeting is by no means a new invention. It's been around for as long as niche publications have existed.

> Fishing pole ads in fishing magazines and motor oil ads in car magazines have existed for hundreds of years.

Reviewer: ```suggestion
Fishing pole ads in fishing magazines and motor oil ads in car magazines have existed for decades.
```

Author: https://www.wired.com/2009/07/dayintech-0730/

The same can be true of software products and solutions. If a developer is using a search engine to find a cloud hosting service,
chances are good that they intend to utilize a cloud hosting service in the near future.
**User Intent** is the [greatest individual factor](https://www.crazyegg.com/blog/guide-user-intent/) to convert a potential candidate into a paying customer.

But search engines are not the only places where software developers exhibit their intent.
From using online tools like [https://jsbin.com](jsbin) rendering their html edits in real-time to scouring package documentation to fix their latest bug,
software developers telegraph their intentions primarily through the content that they are viewing.
This can be numerically demonstrated in the comparison of Click through Rates (CTR) on content-targeted campaigns vs non-content targeted campaigns.

## What our data says

First we'll examine the average weekly CTR across all of our advertising flights that are targeting geographic locations only.
If we had no campaigns ending that week, it's been logged as a 0.
As you can see, almost every week our geo-only campaigns fall below our network average CTR of 0.1%.

> <div class="shadow-lg postimage text-center">
>   <img class="w-75" src="{static}../images/posts/2022-no-keywords-ctr.png" alt="A screenshot of the Click through Rates for untargeted campaigns">

Reviewer: ```suggestion
<div class="postimage text-center">
  <img class="w-75 shadow-lg" src="{static}../images/posts/2022-no-keywords-ctr.png" alt="A screenshot of the Click through Rates for untargeted campaigns">
```

  <p>the average weekly CTR for untargeted campaigns in Q3 and Q4 of 2021</p>
</div>

But how do our content-targeted campaigns fare? Using the same weekly aggregation, it's evident that our content-targeted campaigns frequently exceed our network's average CTR.

> <div class="shadow-lg postimage text-center">
>   <img class="w-75" src="{static}../images/posts/2022-keywords-ctr.png" alt="A screenshot of the Click through Rates for content-targeted campaigns">

Reviewer: ```suggestion
<div class="postimage text-center">
  <img class="w-75 shadow-lg" src="{static}../images/posts/2022-keywords-ctr.png" alt="A screenshot of the Click through Rates for content-targeted campaigns">
```
